core/src/main/java/com/yahoo/ycsb/generator/SequentialGenerator.java

```diff
@@ -0,0 +1,67 @@
+/**                                                                                                                                                                                
+ * Copyright (c) 2010 Yahoo! Inc. All rights reserved.                                                                                                                             
```

Comment:
this line needs updating. If this is an entirely new set of code, then it should just be 2016 YCSB Contributors. If it's a refactoring, then it should be 2010 Yahoo! Inc., 2016 YCSB Contributors.